test/networkpolicy/open_policy.yaml

```diff
@@ -47,10 +47,14 @@ data:
           port: 53
```

Comment:
Shouldn't we delete the open_policy.yaml (and add it to .gitignore) since it has to be copied anyway, depending on the hyperscaler?

Reply:
+1 See f122aa0fb1c32de816e1fbad474a7757048d98d3